test/modules/aws-acm-request-integration.ts

```diff
@@ -53,37 +53,30 @@ describe('AwsAcmRequest Integration Testing', () => {
   `),
   );
 
-  it('installs the acm_request module', install(modules));
+  it('installs the acm module alone', install(['aws_acm']));
 
   it(
-    'adds a new certificate to request',
-    query(`
-    INSERT INTO certificate_request (domain_name)
-    VALUES ('${domainName}');
-  `),
+    'adds a new certificate to request with a domain without route53 support',
+    (done) => {
+      query(`
+        SELECT * FROM certificate_request('fakeDomain.com', 'DNS', '${process.env.AWS_REGION}', '');
+      `)((e: any) => {
+        if (e instanceof Error) {
+          return done();
+        }
+        return done('Somehow did not get an error back from the function');
+      });
+    }
   );
 
-  it('sync before apply (should restore)', sync());
+  it('installs the rest of the modules needed', install(modules));
 
   it(
-    'check no new certificate to request',
-    query(
-      `
-    SELECT *
-    FROM certificate_request;
-  `,
-      (res: any[]) => expect(res.length).toBe(0),
-    ),
-  );
-
-  it(
-    'adds a new certificate to request with incorrect domain',
+    'adds a new certificate to request with a fake domain',
     query(`
-    INSERT INTO certificate_request (domain_name)
-    VALUES ('fakeDomain.com');
-  `),
+      SELECT * FROM certificate_request('fakeDomain.com', 'DNS', '${process.env.AWS_REGION}', '');
+    `, console.log),
```

Comment:
why this `console.log`? Also, isn't this test the same as the one immediately above?

Reply:
Oh, I am surprised the linter didn't catch that. I will remove the debug logging there.

It is repeated, but to test this path failing in two different ways. This one fails "hard" because Route53 is not installed, then we install it and it fails "soft" because AWS cannot validate the fake domain.